codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/protocols/serialize/JsonSerializerGenerator.kt

```diff
@@ -200,23 +200,26 @@ class JsonSerializerGenerator(
         }
     }
 
-    fun renderStructure(
-        writer: RustWriter,
-        structureShape: StructureShape,
-        includedMembers: List<MemberShape>? = null,
-    ) {
+    /*
+    * TODO: expand SdkBody to be also used here. Right now returning it instead of a String requires the server
+    * RestJson1 implementation to be heavily refactored as well as additions to the SdkBody Rust type.
```

Comment:
Proceed with caution on returning SdkBody since it may make event streams harder later. In event streams, you have to serialize protocols into an event stream message rather than to an HTTP request.